x/auth/client/rest/query.go

```diff
@@ -128,19 +133,17 @@ func QueryTxRequestHandlerFn(clientCtx client.Context) http.HandlerFunc {
 			return
 		}
 
-		// We just unmarshalled from Tendermint, we take the proto Tx's raw
-		// bytes, and convert them into a StdTx to be displayed.
-		txBytes := output.Tx.Value
-		stdTx, ok := convertToStdTx(w, clientCtx, txBytes)
-		if !ok {
+		err = packStdTxResponse(w, clientCtx, output)
+		if err != nil {
+			// Error is already returned by packStdTxResponse.
 			return
 		}
 
 		if output.Empty() {
 			rest.WriteErrorResponse(w, http.StatusNotFound, fmt.Sprintf("no transaction found with hash %s", hashHexStr))
 		}
 
-		rest.PostProcessResponseBare(w, clientCtx, stdTx)
+		rest.PostProcessResponseBare(w, clientCtx, output)
```

Comment:
This is on the query tx by hash endpoint. On master: we output a StdTx. In this PR, I changed to output a TxResponse.

This is a breaking change compared to master, but it's actually the same output as 0.39.